services/static-webserver/client/source/class/osparc/auth/ui/RequestAccount.js

```diff
@@ -165,26 +159,22 @@ qx.Class.define("osparc.auth.ui.RequestAccount", {
       const grp = new qx.ui.container.Composite(new qx.ui.layout.VBox(15));
       const buttons = new qx.ui.container.Composite(new qx.ui.layout.HBox(5));
 
-      const eulaTextContainer = new qx.ui.container.Composite(new qx.ui.layout.Flow()).set({
-        width: 100
+      const color = qx.theme.manager.Color.getInstance().resolve("text");
+      const ppText = `I acknowledge  that data will be processed in accordance with <a href='https://sim4life.swiss/privacy' style='color: ${color}' target='_blank''>our privacy policy</a>`;
+      const privacyPolicy = new qx.ui.form.CheckBox().set({
+        required: true,
+        value: false
       })
-      const part1 = "Data will be processed in accordance with"
-      part1.split(" ").forEach(word => eulaTextContainer.add(new qx.ui.basic.Label(word).set({
-        marginRight: 2
-      })))
-      eulaTextContainer.add(new osparc.ui.basic.LinkLabel("our privacy policy.", "https://sim4life.swiss/privacy").set({
-        marginRight: 2,
-        rich: false
-      }))
-      const part2 = "Users are authorized to use the web product in accordance with"
-      part2.split(" ").forEach(word => eulaTextContainer.add(new qx.ui.basic.Label(word).set({
-        marginRight: 2
-      })))
-      eulaTextContainer.add(new osparc.ui.basic.LinkLabel("the EULA.", "https://zurichmedtech.github.io/s4l-manual/#/docs/licensing/copyright_Sim4Life?id=zurich-medtech-ag-zmt").set({
-        rich: false
-      }))
-
-      grp.add(eulaTextContainer)
+      doubleSpaced.push(privacyPolicy);
+      this._form.add(privacyPolicy, ppText, null, "privacyPolicy")
+
+      const eulaText = `I accept the <a href='https://zurichmedtech.github.io/s4l-manual/#/docs/licensing/copyright_Sim4Life?id=zurich-medtech-ag-zmt' style='color: ${color}' target='_blank''>end users license agreement (EULA)</a> and I will use the product in accordance with it.`;
```

Comment:
Should prepare to have something equivalent for osparc.io and s4lite since most likely we will unify request forms

Reply:
👍